instance/tests/models/test_openedx_database_mixins.py

```diff
@@ -524,18 +525,16 @@ def request_overview():
         )
         return response
 
-    def test_rabbitmq_access(self):
-        """
-        Test that RabbitMQ is accessible
-        """
-        response = self.request_overview()
-        self.assertTrue(response.ok)
-        self.assertIn('"protocol":"amqp/ssl"', response.text)
-
+    @responses.activate
     def test_rabbitmq_request(self):
         """
         Test a simple get request with _rabbitmq_request
         """
+        responses.add(
+            method=responses.GET,
+            url='{}/api/overview'.format(settings.RABBITMQ_API_URL),
+            body='{}'
+        )
```

Comment:
This test is no longer verifying anything useful, since it's just testing that the mocked empty `{}` body is the same as the other mocked empty `{}` body.